libminifi/test/capi/CAPITests.cpp

```diff
@@ -33,8 +32,14 @@
 #include <chrono>
 #include <thread>
 
+static nifi_instance *create_instance_obj(const char *name = "random_instance") {
+  nifi_port port;
+  port.port_id = "12345";
+  return create_instance("random_instance", &port);
```

Comment:
How can you set port after this?

Reply:
Are you asking how can they change the port in the instance?

Reply:
My bad, just didn't notice this being in the test code.